cmd/vic-machine/create/create.go

```diff
@@ -138,7 +138,7 @@ func (c *Create) Flags() []cli.Flag {
 		cli.StringFlag{
 			Name:        "client-network, cln",
 			Value:       "",
-			Usage:       "The client network port group name (restricts DOCKER_API access to this network). Defaults to DCHP - see advanced help (-x)",
+			Usage:       "The client network port group inventory path name (restricts DOCKER_API access to this network). Defaults to DCHP - see advanced help (-x)",
```

Comment:
Should this be "inventory path or name"?

Reply:
Yes, I think so, I will change it.